package.json

```diff
@@ -20,13 +20,14 @@
   },
   "scripts": {
     "start": "electron main.js",
-    "test": "powershell provisioning/Tests.ps1 -originalBuildScriptPath './node_modules/gpii-windows/provisioning'; electron tests.js"
+    "test": "powershell provisioning/Tests.ps1 -originalBuildScriptPath './node_modules/gpii-windows/provisioning'; electron tests-universal.js; electron tests-windows.js"
```

Comment:
By separating this into two calls, we wouldn't have a consolidated number of passing tests. Are we ok with that?

Reply:
Well, we aren't ok but I'd say that we can make the effort unless we can ensure that the tests run sequentially.  :smile: The underlying problem is the way we launch the tests, which I guess will change after https://github.com/GPII/gpii-app/pull/17 is ready, isn't it? @michelled @amb26 what do you think?

Reply:
It would be better if I fixed the metrics tests to work.

Reply:
@javihernandez I think we still expect to launch the tests in the same way. Also, I fear that splitting this into two calls will hide failures.